fileredactor/filepathprocessor.go

```diff
@@ -0,0 +1,75 @@
+package fileredactor
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/bitrise-io/go-utils/v2/env"
+	"github.com/bitrise-io/go-utils/v2/pathutil"
+)
+
+// FilePathProcessor is an interface for an entity which accepts file paths separated by the
```

Comment:
It also does env var expansion, I think it should be mentioned in its description.

Reply:
Good catch that env var expansion is actually not needed as the CLI does it. And we are in control of this input so env var expansion will not be disabled. So I have removed that piece of code.